res/default_ddoc_theme.ddoc

```diff
@@ -2,6 +2,8 @@ LPAREN = (
 RPAREN = )
 BACKTICK = `
 DOLLAR = $
+COMMA = ,
+QUOTE = &quot;
```

Comment:
Requires an update to:
- https://github.com/dlang/dlang.org/blob/master/verbatim.ddoc
- https://github.com/dlang/dlang.org/blob/master/spec/latex.ddoc

Reply:
This definition seems out of place. All others are universal - plain paren, plain backtick, plain dollar. This is HTML-specific so its hould go in an HTML-specific file. Granted, all others work out in HTML...

Reply:
I'll remove the `QUOTE` macro until it's needed, which is for URIs. It is indeed specific to HTML, but will have to be one of the built-in macros because it'll be generated by the ddoc processor.